cmd/goatcounter/serve.go

```diff
@@ -81,6 +81,11 @@ Flags:
   -public-port Port your site is publicly accessible on. Only needed if it's
                not 80 or 443.
 
+  -base-path   Base path to use in URLs sent back to the client. Default: "/".
+               You need this if you are reverse proxying GoatCounter into a
+               subdirectory of your website. This does not affect routing, since
+               it is assumed the reverse proxy will strip off the base path.
```

Comment:
This text can probably be a bit easier to understand:

> Path under which GoatCounter is available. Usually GoatCounter runs on its own domain or subdomain ("stats.example.com"), but in some cases it's useful to run GoatCounter under a path ("example.com/stats"), in which case you'll beed to set this to "/stats".

And it should probably update the routing as well, unless that's a ton of effort? I think that would be a lot clearer.

Reply:
Yeah that makes sense. I don't think it should be too hard, I'll give it a shot. I agree it's clearer, and anyone in a position to configure their reverse proxy should be able to format the path how they like (i.e. not strip off the prefix).

Reply:
> in which case you'll beed to set this to "/stats".

You copied my "beed" typo 😅

Reply:
Fixed :)